cpp/src/parquet/metadata.h

```diff
@@ -182,6 +184,28 @@ class PARQUET_EXPORT ColumnChunkMetaData {
   std::unique_ptr<ColumnChunkMetaDataImpl> impl_;
 };
 
+// \brief DataPageStats stores statistics about a data page including number of
+// values and rows.
+class PARQUET_EXPORT DataPageStats {
+ public:
+  explicit DataPageStats(const EncodedStatistics& encoded_statistics, int32_t num_values,
```

Comment:
document encoded_statistics must live longer then this object (or maybe take it as a pointer to make this slightly more explicit).

Reply:
Done